config/routes.rb

```diff
@@ -1,10 +1,24 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-  root 'application#index'
+  resources :attractions, only: [:index]
+
+  resources :users
+
+  resources :home
+
+  get '/users/:id/categories' => 'users#travel_profile', as: :travel_profile
```

Comment:
You can do this inside the `resources :users` block [using member/collection](http://guides.rubyonrails.org/routing.html#adding-more-restful-actions).

```ruby
resource :users do
  get 'categories', to: 'users#travel_profile`, on: :member, as: :travel_profile
end
```